config/routes.rb

```diff
@@ -14,6 +14,8 @@
 
   resources :adventures, only: [:new, :create, :index]
 
+  patch '/adventures/:id/edit', to: 'adventures#update'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.